Neo4j.Driver/Neo4j.Driver.Tests/Internal/Protocol/BoltProtocolV3Tests.cs

```diff
@@ -385,14 +389,15 @@ public async Task ShouldThrowIfImpersonatedUserIsNotNull()
             {
                 var mockConn = new Mock<IConnection>();
                 mockConn.SetupGet(x => x.Version).Returns(BoltProtocolVersion.V3_0);
+                mockConn.SetupGet(x => x.SessionConfig).Returns(new SessionConfig("Douglas Fire"));
 
                 var exception = await Record.ExceptionAsync(
                     () => BoltProtocolV3.Instance.BeginTransactionAsync(
                         mockConn.Object,
                         null,
                         null,
                         TransactionConfig.Default,
-                        "Douglas Fire",
+                        null,
```

Comment:
we really need to remove these values from the method signatures if they are being pulled from the session config now, this could lead to bugs etc.